audio_service/lib/audio_service.dart

```diff
@@ -246,8 +246,7 @@ class PlaybackState {
     this.shuffleMode = AudioServiceShuffleMode.none,
     this.captioningEnabled = false,
     this.queueIndex,
-  })  : assert(androidCompactActionIndices == null ||
-            androidCompactActionIndices.length <= 3),
+  })  : assert(androidCompactActionIndices == null || androidCompactActionIndices.length <= 3),
```

Comment:
Is there any substantive change here? If it's only an indentation change, please stick with `dart format` which is what I used, and it will help to prevent the diff from showing things that aren't actually different.

Reply:
Ok.
Will work on it and remove formate changes in PR.